config.py.sample

```diff
@@ -0,0 +1,4 @@
+import os
+
+class Config(object):
+    SECRET_KEY = os.environ.get('SECRET_KEY') or 'dummy'
```

Comment:
Add all the configs with default values in the sample. I am sure you added more configurations like `SQLAlchemy` etc.

Reply:
Done